src/pages/QueueStatus/Components/Queues.js

```diff
@@ -0,0 +1,263 @@
+`;
+
+const QueueWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  & .circle {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 50%;
+    width: ${props => (props.isDesktop ? "100px" : "35px")};
+    height: ${props => (props.isDesktop ? "100px" : "35px")};
+    background-color: #4c0788;
+    }
+    & p {
+      font-size: ${props => (props.isDesktop ? "33px" : "12px")};
+      font-weight: 900;
+      color: white;
+      margin: 0;
+    }
+  }
+  & .circle-blink {
+    @keyframes blink {
+      50% {
+        background-color: #ffc836;
+        & p {
+          color: #4c0788;
+        }
+      }
+    }
+    @-webkit-keyframes blink {
+      50% {
+        background-color: #ffc836;
+        & p {
+          color: #4c0788;
+        }
+      }
+    }
+    animation-duration: 600ms;
+    animation-name: "blink";
+    animation-direction: normal;
+    -webkit-animation: blink 600ms;
+  }
+  & .name {
+    display: flex;
+    margin-left: -30px;
+    padding-left: 45px;
+    padding-right: 30px;
+    align-items: center;
+    line-height: 1.2;
+    width: 60%;
+    height: ${props => (props.isDesktop ? "90px" : "35px")};
+    background-color: #e8e8e8;
+    z-index: -1;
+    border-radius:${props => !props.isDesktop && "40px"};
+    border-top-right-radius: 40px;
+    border-bottom-right-radius: 40px;
+    & p {
+      font-size: ${props => (props.isDesktop ? "3vh" : "10px")} ;
+      font-weight: 900;
+      color: #4c0788;
+      margin: 0;
+      word-break: break-all;
+    }
+  }
+`;
+
+const GridArea = styled.div`
+  display: grid;
+  width: 100%;
+  height: 100%;
+  grid-template-columns: repeat(2, minmax(100px, 1fr));
+  grid-template-rows: ${props =>
+    props.isDesktop
+      ? "repeat(auto-fill, minmax(140px, 1fr))"
+      : "repeat(auto-fill, minmax(70px, 1fr))"};
+  column-gap: 10px;
+`;
+
+const BigLabel = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 40px;
+  border-radius: 55px;
+  max-width: 500px;
+  width:  ${props => (props.isDesktop ? "100%" : "90%")};
+  height: ${props => (props.isDesktop ? "100px" : "60px")} ;
+  padding:1rem;
+  margin-bottom: 100px;
+  background-color: #e4e4e4;
+  & p {
+  font-size:${props => (props.isDesktop ? "50px" : "20px")};
+      margin 0;
+      font-weight: 900;
+  }
+`;
+
+const BigCircle = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: ${props => (props.isDesktop ? "320px" : "100px")};
+  height: ${props => (props.isDesktop ? "320px" : "100px")};
+  border-radius: 100%;
+  background-color: #ffc836;
+  & p {
+    color: white;
+  font-size: ${props => (props.isDesktop ? "95px" : "35px")};
+    font-weight: 900;
+    margin 0;
+}
+`;
+
+const handleClassUpdate = (queueId, updatedQueue, setUpdatedQueue) => {
+  setTimeout(() => {
+    setUpdatedQueue("");
+  }, 610);
```

Comment:
I know it's not related to this PR, but I think the timeout value could be increased @luismendes535.